packages/terra-action-header/tests/wdio/action-header-spec.js

```diff
@@ -14,9 +14,7 @@ describe('ActionHeader', () => {
         '--terra-action-header-border-left': '1px solid #00FF00',
         '--terra-action-header-border-right': '1px solid #00FF00',
         '--terra-action-header-border-top': '1px solid #00FF00',
-        '--terra-action-header-box-shadow': '0 1px 0 0 #FF8C00',
-        '--terra-action-header-font-color': '#4682B4',
-        '--terra-action-header-font-family': 'Roboto',
+        '--terra-action-header-box-shadow': '5px 5px 5px #551A8B',
```

Comment:
This screenshot should have been updated but I doesn't look like it took

Reply:
Discussed this with @bjankord, this might be a bug. I think we are going to move forward without the fix for now.

Reply:
Out of curiosity, are you able to apply a box-shadow and see the changes locally? If the box shadow is being applied to the outside of the header component itself, its likely that the screenshot test is not capturing it because the box shadow is outside the element itself. One thing to try would be applying an inset-box shadow.